### PR TITLE
update readme file about Dr.Derc-large model

### PR DESCRIPTION
**Pull request by yulongl-ibm**
# Pull Request      

## What does this PR do?

```Closes #(369)```       

Notes:
- Replace `(issue)` **above** ↑↑↑ with the issue this PR closes to automatically link the two.
This must be done when the PR is created.
- Add multiple `Closes #(issue)` as needed.
- If this PR is work towards but does not close an issue, simply tag the issue without mentioning `Closes`.

## Description

**Describe** the changes proposed by this PR below to give the reviewer context below ↓↓↓

```
1. Updated DrDecr readme file including description and parameters to train the large model
2. Upload a screenshot of XOR-TyDi leaderboard
```     

### Request Review

Be sure to **request a review** from one or more reviewers (unless the PR is to an unprotected branch).

## Versioning

When opening a PR to make changes to PrimeQA (i.e. `primeqa/`) master, be sure to increment the version following
[semantic versioning](https://semver.org/).  The VERSION is stored [here](https://github.com/primeqa/primeqa/blob/main/VERSION) 
and is incremented using `bump2version {patch,minor,major}` as described in the (development guide documentation)[development.html].

 - [ ] Have you updated the VERSION?
 - [ ] Or does this PR not change the `primeqa` package or was not into master?

After pulling in changes from master to an existing PR, ensure the VERSION is updated appropriately.
This may require bumping the version again if it has been previously bumped.

If you're not quite ready yet to post a PR for review, feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

### After Merging
If merging into master and VERSION was updated, after this PR is merged:
- [ ] [Create a release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) from the master with version equal to [VERSION](https://github.com/primeqa/primeqa/blob/main/VERSION)
- [ ] Not merging to master or VERSION not updated

### Checklist

Review the following and mark as completed:
- [ ] [Tag an issue or issues](#what-does-this-pr-do) this PR addresses.
- [ ] Added [description](#description) of changes proposed.
- [ ] Review requested as [appropriate](#request-review).
- [ ] Version bumped as [appropriate](#Versioning).
- [ ] New classes, methods, and functions documented.
- [ ] Documentation for modified code is updated.
- [ ] Built documentation to confirm it renders as expected (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Tests added to ensure all functionalities tested at >= 60% unit test coverage (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Release created [as needed](#after-merging) after merging.

### CODE REVIEW

**Review by franzmpub — APPROVED** [review]
I made a few changes in parameter names to reflect the current version used in the tools.
LGTM

**Review by bhavani105 — APPROVED** [review]
LGTM

File: examples/drdecr/README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 This file provides instructions on how to train and test the Dr. DECR models (Dense Retrieval with Distillation-Enhanced Cross-Lingual Representation) Student/Teacher pipeline, as desribed in [Learning Cross-Lingual IR from an English Retriever](https://arxiv.org/abs/2112.08185). 
 
-Similar models based on ColBERT V1 were used to obtain the results in the Dr. DECR XOR-TyDi leaderboard whitebox (not using external APIs) submission.
+As of 11/8/2022, Dr. Decr-large is the best model on [XOR-TyDi](https://nlp.cs.washington.edu/xorqa/) leaderboard.
```
Comment (seirasto) [inline examples/drdecr/README.md:5]:
```suggestion
As of 11/8/2022, Dr. DECR-large is the best model on the [XOR-TyDi](https://nlp.cs.washington.edu/xorqa/) leaderboard.
```

**Review by jdpsen — APPROVED** [review]
LGTM.

**Pull request merged**